sdk/resourcemanager/automanage/armautomanage/README.md

```diff
@@ -49,9 +61,127 @@ options := arm.ClientOptions {
         Cloud: cloud.AzureChina,
     },
 }
-client, err := armautomanage.NewReportsClient(<subscription ID>, cred, &options)
+reportsClient, err := armautomanage.NewReportsClient("<subscription ID>", cred, &options)
+```
+
+## Create or Update a Custom Automanage Configuration Profile
+
+To update a profile, provide a value for all properties as if you were creating a configuration profile (ID, Name, Type, Location, Properties, Tags)
+
+```go
+configuration := map[string]interface{}{
+    "Antimalware/Enable":                false,
+    "AzureSecurityCenter/Enable":        true,
+    "Backup/Enable":                     false,
+    "BootDiagnostics/Enable":            true,
+    "ChangeTrackingAndInventory/Enable": true,
+    "GuestConfiguration/Enable":         true,
+    "LogAnalytics/Enable":               true,
+    "UpdateManagement/Enable":           true,
+    "VMInsights/Enable":                 true,
+}
+
+properties := armautomanage.ConfigurationProfileProperties{
+    Configuration: configuration,
+}
+
+id := "/subscriptions/<subscription ID>/resourceGroups/resourceGroupName/providers/Microsoft.Automanage/configurationProfiles/configurationProfileName"
+resourceType := "Microsoft.Automanage/configurationProfiles"
+location := "eastus"
+environment := "dev"
+configurationProfileName := "name"
+
+// tags may be omitted 
+tags := make(map[string]*string)
+tags["environment"] = &environment
+
+newProfile := armautomanage.ConfigurationProfile{
+    ID:         &id,
+    Name:       &configurationProfileName,
+    Type:       &resourceType,
+    Location:   &location,
+    Properties: &properties,
+    Tags:       tags,
+}
+
+configProfilesClient.CreateOrUpdate(context.Background(), configurationProfileName, "resourceGroupName", newProfile, nil)
+```
+
+
+## Get an Automanage Configuration Profile
+
+```go
+profile, err := configProfilesClient.Get(context.Background(), "configurationProfileName", "resourceGroupName", nil)
+data, err := json.MarshalIndent(profile, "", "   ")
+
+fmt.Println(string(data))
+```
+
+
+## Delete an Automanage Configuration Profile
+
+```go
+configProfilesClient.Delete(context.Background(), "resourceGroupName", "configurationProfileName", nil)
```

Comment:
```suggestion
err := configProfilesClient.Delete(context.Background(), "resourceGroupName", "configurationProfileName", nil)
```

Reply:
Done. 

I have "_, err := configProfilesClient.Delete" because this method only returns an empty object if there is a successful delete.